yio-model-mediaplayer.pri

```diff
@@ -7,8 +7,10 @@ INCLUDEPATH += $$PWD/src
 # Model files
 HEADERS += \
     $$PWD/src/yio-model/mediaplayer/albummodel_mediaplayer.h \
-    $$PWD/src/yio-model/mediaplayer/searchmodel_mediaplayer.h
+    $$PWD/src/yio-model/mediaplayer/searchmodel_mediaplayer.h \
+    $$PWD/src/yio-model/mediaplayer/tvchannelmodel_mediaplayer.h
```

Comment:
New header file is missing in pull request.

Reply:
files added